tests/console/zypper_info.pm

```diff
@@ -18,19 +18,36 @@ use testapi;
 use utils;
 
 sub run() {
+
+    my $info_output;
+    my $expected_header;
+    my $expected_package_name;
+
     select_console 'root-console';
 
-    zypper_call('mr -e repo-source');
-    zypper_call('ref');
-    my $info_output = script_output 'zypper info srcpackage:htop';
+    $info_output = script_output 'zypper info vim';
 
-    my $expected_header = 'Information for srcpackage htop:';
+    $expected_header = 'Information for package vim:';
     die "Missing info header. Expected: /$expected_header/"
       unless $info_output =~ /$expected_header/;
 
-    my $expected_package_name = 'Name                  : htop';
+    $expected_package_name = 'Name *: vim';
     die "Missing package name. Expected: /$expected_package_name/"
       unless $info_output =~ /$expected_package_name/;
+
+    if (check_var('DISTRI', 'opensuse')) {
```

Comment:
It is not obvious to the reader why not the same for SLE. I know but the source code should tell.